.github/workflows/verify-commit.yml

```diff
@@ -0,0 +1,29 @@
+name: Verify Commit
+
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+
+permissions:
+  actions: read
+  contents: read
+
+jobs:
+  main:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 8
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
```

Comment:
what is the current Node version for the app? 
My recommendation would be `v20.13.1` as its currently LTS